addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -92,4 +92,68 @@ tour.register('edit_megamenu', {
         run: function () {}, // it's a check
     },
 ]);
+tour.register('edit_megamenu_big_icons_subtitles', {
+    test: true,
+    url: '/?enable_editor=1',
+}, [
+    // Add a megamenu item to the top menu.
+    {
+        content: "Click on a menu item",
+        trigger: '#top_menu .nav-item a',
+    },
+    {
+        content: "Click on 'Link' to open Link Dialog",
+        trigger: '.o_edit_menu_popover a.js_edit_menu',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        extra_trigger: ".o_web_editor_dialog:visible",
+        trigger: '.modal-body a.js_add_menu[data-type="mega"]',
+    },
+    {
+        content: "Write a label for the new menu item",
+        extra_trigger: ".o_link_dialog",
+        trigger: '.o_link_dialog #o_link_dialog_label_input',
+        run: 'text Megaaaaa!',
+    },
+    {
+        content: "Save the new menu item",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    {
+        content: "Save the changes to the menu",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    // Edit a menu item
+    wTourUtils.clickOnEdit(),
+    wTourUtils.clickOnExtraMenuItem({extra_trigger: '#oe_snippets.o_loaded'}),
```

Comment:
Could you add unit tests to cover these flows in a more targeted way?

Reply:
The reason for not writing the unit test in the first place was, that the fix is in `destroyLinkTools` and it is triggered when clicking on the side toolbar. I couldn't find a way to access the side toolbar on website or in mass_mailing's QUnit's, I could only find tours when side toolbar are used.

Reply:
If it's specific to the sidebar then indeed this is probably your best bet.